tests/test_responses.py

```diff
@@ -309,6 +312,37 @@ async def app(scope, receive, send):
     assert response.text == "Hello, world!"
 
 
+def test_set_cookie_with_expire_datetime(test_client_factory):
+    async def app(scope, receive, send):
+        response = Response("Hello, world!", media_type="text/plain")
+        now = datetime.now(timezone.utc)
+        expires = now + timedelta(minutes=1)
+        response.set_cookie(
+            "mycookie",
+            "myvalue",
+            max_age=10,
+            expires=expires,
+            path="/",
+            domain="localhost",
+            secure=True,
+            httponly=True,
+            samesite="none",
```

Comment:
Can we set only the fields we are interested i.e. `expires`, `key` and `value` here?